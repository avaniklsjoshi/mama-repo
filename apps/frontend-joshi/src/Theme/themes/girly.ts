export default {
  themeType: "girly",
  colors: {
    background: "#f1b7ea",
    text: "#fff"
  },
  header: {
    background: "#000",
    text: "#fff"
  },
  footer: {
    background: "#262726",
    text: "#fff"
  },
  section: {
    background: "#008F7A",
    text: "#fff"
  }
};
