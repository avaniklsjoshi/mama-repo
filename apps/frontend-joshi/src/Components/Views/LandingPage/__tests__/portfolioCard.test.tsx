/* eslint-disable testing-library/no-node-access */
/* eslint-disable testing-library/no-container */
import { act, fireEvent, render, screen } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import girly from "../../../../Theme/themes/girly";
import PortfolioCard from "../portfolioCard";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key: any) => key })
}));

describe("Landing Page", () => {
  it("renders Portfolio Card", () => {
    const { container } = render(
      <ThemeProvider theme={girly}>
        <PortfolioCard />
      </ThemeProvider>
    );
    expect(container).toMatchSnapshot();
    expect(screen.getByText("fullName")).toBeInTheDocument();
  });
});
