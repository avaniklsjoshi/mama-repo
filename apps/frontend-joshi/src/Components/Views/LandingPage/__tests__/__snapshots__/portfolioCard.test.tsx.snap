// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page renders Portfolio Card 1`] = `
<div>
  <div
    class="portfolio-container"
  >
    <div
      class="card card"
    >
      <div
        class="border"
      >
        <div
          class="card-name"
        >
          fullName
        </div>
        <div
          class="icons"
        >
          <div>
            <a
              aria-label="Twitter Avani"
              class="social-links"
              href="https://twitter.com/avaniklsjoshi/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                height="24px"
                id="Capa_1"
                version="1.1"
                viewBox="0 0 512 512"
                width="24px"
                x="0px"
                y="0px"
              >
                <path
                  d="M512,97.248c-19.04,8.352-39.328,13.888-60.48,16.576c21.76-12.992,38.368-33.408,46.176-58.016  c-20.288,12.096-42.688,20.64-66.56,25.408C411.872,60.704,384.416,48,354.464,48c-58.112,0-104.896,47.168-104.896,104.992  c0,8.32,0.704,16.32,2.432,23.936c-87.264-4.256-164.48-46.08-216.352-109.792c-9.056,15.712-14.368,33.696-14.368,53.056  c0,36.352,18.72,68.576,46.624,87.232c-16.864-0.32-33.408-5.216-47.424-12.928c0,0.32,0,0.736,0,1.152  c0,51.008,36.384,93.376,84.096,103.136c-8.544,2.336-17.856,3.456-27.52,3.456c-6.72,0-13.504-0.384-19.872-1.792  c13.6,41.568,52.192,72.128,98.08,73.12c-35.712,27.936-81.056,44.768-130.144,44.768c-8.608,0-16.864-0.384-25.12-1.44  C46.496,446.88,101.6,464,161.024,464c193.152,0,298.752-160,298.752-298.688c0-4.64-0.16-9.12-0.384-13.568  C480.224,136.96,497.728,118.496,512,97.248z"
                  fill="#03A9F4"
                />
              </svg>
            </a>
            <a
              aria-label="LinkedIn Avani"
              class="social-links"
              href="https://www.linkedin.com/in/avani-joshi-6365b887/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                enable-background="new 0 0 24 24"
                height="24px"
                viewBox="0 0 24 24"
                width="24px"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#0077b5"
                >
                  <path
                    d="m23.994 24v-.001h.006v-8.802c0-4.306-.927-7.623-5.961-7.623-2.42 0-4.044 1.328-4.707 2.587h-.07v-2.185h-4.773v16.023h4.97v-7.934c0-2.089.396-4.109 2.983-4.109 2.549 0 2.587 2.384 2.587 4.243v7.801z"
                  />
                  <path
                    d="m.396 7.977h4.976v16.023h-4.976z"
                  />
                  <path
                    d="m2.882 0c-1.591 0-2.882 1.291-2.882 2.882s1.291 2.909 2.882 2.909 2.882-1.318 2.882-2.909c-.001-1.591-1.292-2.882-2.882-2.882z"
                  />
                </g>
              </svg>
            </a>
            <a
              aria-label="github Avani"
              class="social-links"
              href="https://github.com/avaniklsjoshi"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                enable-background="new 0 0 24 24"
                height="24px"
                viewBox="0 0 24 24"
                width="24px"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="m12 .5c-6.63 0-12 5.28-12 11.792 0 5.211 3.438 9.63 8.205 11.188.6.111.82-.254.82-.567 0-.28-.01-1.022-.015-2.005-3.338.711-4.042-1.582-4.042-1.582-.546-1.361-1.335-1.725-1.335-1.725-1.087-.731.084-.716.084-.716 1.205.082 1.838 1.215 1.838 1.215 1.07 1.803 2.809 1.282 3.495.981.108-.763.417-1.282.76-1.577-2.665-.295-5.466-1.309-5.466-5.827 0-1.287.465-2.339 1.235-3.164-.135-.298-.54-1.497.105-3.121 0 0 1.005-.316 3.3 1.209.96-.262 1.98-.392 3-.398 1.02.006 2.04.136 3 .398 2.28-1.525 3.285-1.209 3.285-1.209.645 1.624.24 2.823.12 3.121.765.825 1.23 1.877 1.23 3.164 0 4.53-2.805 5.527-5.475 5.817.42.354.81 1.077.81 2.182 0 1.578-.015 2.846-.015 3.229 0 .309.21.678.825.56 4.801-1.548 8.236-5.97 8.236-11.173 0-6.512-5.373-11.792-12-11.792z"
                  fill="#212121"
                />
              </svg>
            </a>
            <a
              aria-label="gitlab Avani"
              class="social-links"
              href="https://gitlab.com/avaniklsjoshi"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                height="24px"
                version="1.1"
                viewBox="0 0 210 194"
                width="24px"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                  id="Page-1"
                  stroke="none"
                  stroke-width="1"
                >
                  <g
                    id="Fill-1-+-Group-24"
                  >
                    <g
                      id="Group-24"
                    >
                      <g
                        id="Group"
                      >
                        <path
                          d="M105.0614,193.655 L105.0614,193.655 L143.7014,74.734 L66.4214,74.734 L105.0614,193.655 L105.0614,193.655 Z"
                          fill="#E24329"
                          id="Fill-4"
                        />
                        <path
                          fill="#FC6D26"
                          id="Fill-6"
                        />
                        <path
                          d="M105.0614,193.6548 L66.4214,74.7338 L12.2684,74.7338 L105.0614,193.6548 Z"
                          fill="#FC6D26"
                          id="Fill-8"
                        />
                        <path
                          fill="#FC6D26"
                          id="Fill-10"
                        />
                        <path
                          d="M12.2685,74.7341 L12.2685,74.7341 L0.5265,110.8731 C-0.5445,114.1691 0.6285,117.7801 3.4325,119.8171 L105.0615,193.6551 L12.2685,74.7341 Z"
                          fill="#FCA326"
                          id="Fill-12"
                        />
                        <path
                          fill="#FC6D26"
                          id="Fill-14"
                        />
                        <path
                          d="M12.2685,74.7342 L66.4215,74.7342 L43.1485,3.1092 C41.9515,-0.5768 36.7375,-0.5758 35.5405,3.1092 L12.2685,74.7342 Z"
                          fill="#E24329"
                          id="Fill-16"
                        />
                        <path
                          d="M105.0614,193.6548 L143.7014,74.7338 L197.8544,74.7338 L105.0614,193.6548 Z"
                          fill="#FC6D26"
                          id="Fill-18"
                        />
                        <path
                          d="M197.8544,74.7341 L197.8544,74.7341 L209.5964,110.8731 C210.6674,114.1691 209.4944,117.7801 206.6904,119.8171 L105.0614,193.6551 L197.8544,74.7341 Z"
                          fill="#FCA326"
                          id="Fill-20"
                        />
                        <path
                          d="M197.8544,74.7342 L143.7014,74.7342 L166.9744,3.1092 C168.1714,-0.5768 173.3854,-0.5758 174.5824,3.1092 L197.8544,74.7342 Z"
                          fill="#E24329"
                          id="Fill-22"
                        />
                      </g>
                    </g>
                  </g>
                </g>
              </svg>
            </a>
            <button
              aria-label="ðŸŽ‰ Link Copied!"
              class="share-button"
              data-mui-internal-clone-element="true"
            >
              <svg
                height="24px"
                viewBox="0 0 64 64"
                width="24px"
                xmlns="http://www.w3.org/2000/svg"
              >
                <lineargradient
                  gradientUnits="userSpaceOnUse"
                  id="qleorMOLqjiXYpxSjD5yKd"
                  spreadMethod="reflect"
                  x1="32.001"
                  x2="32.001"
                  y1="53.501"
                  y2="10.5"
                >
                  <stop
                    offset="0"
                    stop-color="#37ab6a"
                  />
                  <stop
                    offset=".422"
                    stop-color="#39b66f"
                  />
                  <stop
                    offset="1"
                    stop-color="#3ac074"
                  />
                </lineargradient>
                <path
                  d="M48.996,53.501c-0.676,0-1.362-0.153-2.009-0.476l-34-17C11.463,35.263,10.5,33.704,10.5,32 c0-1.704,0.963-3.263,2.487-4.025l34-17c2.226-1.109,4.926-0.21,6.038,2.013c1.111,2.223,0.21,4.926-2.013,6.038L25.062,32 l25.95,12.975c2.223,1.112,3.124,3.815,2.013,6.038C52.236,52.589,50.647,53.501,48.996,53.501z"
                  fill="url(#qleorMOLqjiXYpxSjD5yKd)"
                />
                <lineargradient
                  gradientUnits="userSpaceOnUse"
                  id="qleorMOLqjiXYpxSjD5yKe"
                  spreadMethod="reflect"
                  x1="15"
                  x2="15"
                  y1="23"
                  y2="41"
                >
                  <stop
                    offset="0"
                    stop-color="#62de8f"
                  />
                  <stop
                    offset=".478"
                    stop-color="#5dd18d"
                  />
                  <stop
                    offset="1"
                    stop-color="#56be89"
                  />
                </lineargradient>
                <path
                  d="M24,32c0,4.972-4.028,9-9,9s-9-4.028-9-9s4.028-9,9-9S24,27.028,24,32z"
                  fill="url(#qleorMOLqjiXYpxSjD5yKe)"
                />
                <lineargradient
                  gradientUnits="userSpaceOnUse"
                  id="qleorMOLqjiXYpxSjD5yKf"
                  spreadMethod="reflect"
                  x1="49"
                  x2="49"
                  y1="6"
                  y2="24"
                >
                  <stop
                    offset="0"
                    stop-color="#62de8f"
                  />
                  <stop
                    offset=".478"
                    stop-color="#5dd18d"
                  />
                  <stop
                    offset="1"
                    stop-color="#56be89"
                  />
                </lineargradient>
                <path
                  d="M58,15c0,4.972-4.028,9-9,9s-9-4.028-9-9s4.028-9,9-9S58,10.028,58,15z"
                  fill="url(#qleorMOLqjiXYpxSjD5yKf)"
                />
                <lineargradient
                  gradientUnits="userSpaceOnUse"
                  id="qleorMOLqjiXYpxSjD5yKg"
                  spreadMethod="reflect"
                  x1="49"
                  x2="49"
                  y1="40"
                  y2="58"
                >
                  <stop
                    offset="0"
                    stop-color="#62de8f"
                  />
                  <stop
                    offset=".478"
                    stop-color="#5dd18d"
                  />
                  <stop
                    offset="1"
                    stop-color="#56be89"
                  />
                </lineargradient>
                <path
                  d="M58,49c0,4.972-4.028,9-9,9s-9-4.028-9-9s4.028-9,9-9S58,44.028,58,49z"
                  fill="url(#qleorMOLqjiXYpxSjD5yKg)"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
